product_docs/docs/migration_toolkit/55/06_building_toolkit.properties_file.mdx

```diff
@@ -113,7 +113,7 @@ The URL contains the following information:
 
 -   `{SRC_DB_USER|TARGET_DB_USER}` — Specify a user with privileges to create each type of object migrated. If migrating data into a table, the specified user might also need insert, truncate, and references privileges for each target table.
 
--   `{SRC_DB_PASSWORD|TARGET_DB_PASSWORD}` — Set to the password of the privileged PostgreSQL user.
+-   `{SRC_DB_PASSWORD|TARGET_DB_PASSWORD}` — Set to the password of the privileged PostgreSQL user. Do not wrap the password in quotes.
```

Comment:
I would suggest to use "Provide the password without any surrounding quotes." to keep it consistent with earlier modifications you did above.